DQM/DTMonitorModule/python/dtDQMOfflineSources_Cosmics_cff.py

```diff
@@ -42,11 +46,12 @@
 from DQM.DTMonitorModule.dtTriggerEfficiencyTask_cfi import *
 
 dtSourcesCosmics = cms.Sequence(dtDataIntegrityUnpacker  +
-                                DTDataIntegrityTask +
-                                dtDCSByLumiMonitor + 
-                                dtRunConditionVar + 
-                                dtSegmentAnalysisMonitor +
-                                dtResolutionAnalysisMonitor +
-                                dtEfficiencyMonitor +
-                                dtTriggerEfficiencyMonitor +
-                                dqmInfoDT)
+                         twinMuxStage2Digis +
```

Comment:
I am not sure what David meant. L1 reco sequence? To me, dtDQMOfflineSources_*.py looks a proper place for it. Ping, @davidlange6 @rekovic